readthedocs/core/static-src/core/js/doc-embed/version-compare.js

```diff
@@ -25,11 +25,14 @@ function init(data) {
       .attr('href', currentURL)
       .text(data.slug);
 
-    var body = $("div.body");
-    if (!body.length) {
-        body = $("div.document");
+    var selectors = ['[role=main]', 'main', 'div.body', 'div.document'];
+    for (var i = 0; i < selectors.length; i += 1) {
```

Comment:
I think we should copy these lines to the `sphinx-warning-banner` extension. Currently, it only uses one selector and the user can change it via a config, but I suppose we could try these same different selectors by default: https://github.com/humitos/sphinx-version-warning/blob/master/versionwarning/_static/js/versionwarning.src.js#L21

Reply:
https://github.com/humitos/sphinx-version-warning/issues/32